### DISCUSSION TITLE
Check multiple permissions

### DISCUSSION THREAD

**Discussion by matoous** [Permify/permify#621] [Q&A]
Hi!

Is there a way to check multiple permissions at once? We have this specific use-case: to render appropriate UI we want to check whether the user has specific set of permissions. The Subject Permission List API can return full set of permissions the Subject has in specific Entity, in theory we could filter-out only those that are requested by the UI application but maybe this is more expensive as compared to checking only the permissions that are specifically requested?

**Comment by Charlotte1959** [comment]
Approved

**Comment by thisisnkc** [comment]
Hi @matoous this is merged, see this [PR](https://github.com/Permify/permify/pull/2657) for reference.
